plugins/action/action_result.h

```diff
@@ -14,7 +14,7 @@ namespace dronecore {
 enum class ActionResult {
```

Comment:
Yeah it is actually here. Why do you have another `Result` in `Action`, then?

Reply:
Probably re-appeared in a merge?

Reply:
Yeah I did not formulate it in a good way. The other one should be removed xD.